### PR TITLE
New ChIP-seq tutorial

### PR DESCRIPTION
**Pull request by malloryfreeberg**
Created new MD-style tutorial for ChIP-seq analysis of the TAL1 TF in mouse hematopoietic cell lines. Update includes screenshot images of workflow.

### CODE REVIEW

**Comment by friedue** [comment]
so, @bebatut, should I merge this? I like it a lot!

File: ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ Identification of the binding sites of the Estrogen receptor with ChIP-sequencin
 
 - *How can we analyze raw ChIP-seq data?*
 - *What are the binding sites of the Estrogen receptor?*
-- *Third question*
+- *Which genes appear to be regulated by the Estrogen receptor?*
```
Comment (bebatut) [inline ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md]: What does this file contain? If it is empty, could be better to remove it :)

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+- *Determine unique/common Tal1 binding sites from G1E and Megakaryocytes*
+- *Identify unique/common Tal1 peaks occupying gene promoters*
+- *Visually inspect Tal1 peaks with Trackster*
+
+:heavy_check_mark: ***Requirements***
+
+- *Galaxy introduction* (add link)
+- *NGS-QC* (add [link](../../NGS-QC/slides/dive_into_qc.md))
+- *NGS-mapping* (add [link](../../NGS-mapping/slides/dive_into_mapping.md))
+- *Trackster* (add link)
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: The link is missing

**Comment by bgruening** [comment]
@friedue please feel free to merge!
Thanks a lot @malloryfreeberg!

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+2. Import the ChIP-seq raw data (\*.fastqsanger) from Zenodo [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.197100.svg)](https://doi.org/10.5281/zenodo.197100)
+
+
+    - In Zenodo: Right-click a FASTQ filename → Copy Link Address
+    - In Galaxy: Get Data → Upload File from your computer → Paste/Fetch data
+    - Paste the copied link into the dialog box and set the datatype to 'fastqsanger'
+    - Click Start
+    - Repeat for all FASTQ files
+
+    <img src="../images/upload_data_page.png"/>
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: You can add figure directly in markdown: ``

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+
+
+    - In Zenodo: Right-click a FASTQ filename → Copy Link Address
+    - In Galaxy: Get Data → Upload File from your computer → Paste/Fetch data
+    - Paste the copied link into the dialog box and set the datatype to 'fastqsanger'
+    - Click Start
+    - Repeat for all FASTQ files
+
+    <img src="../images/upload_data_page.png"/>   
+    <figcaption><b>Figure X:</b> Data can be imported directly with links.</figcaption>
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: I am not sure if the caption is really needed if the figure is introduced before. And if we conserve the caption, maybe removing the "Figure X". It will be difficult to keep up to date if the tutorial changes

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 5]
```diff
@@ -0,0 +1,420 @@
...
+  - [Step 5: Assessing GC bias](#step5)
+  - [Step 6: Assessing IP strength](#step6)
+  - [Step 7: Determining Tal1 binding sites](#step7)
+  - [Step 8: Inspection of Tal1 peaks](#step8)
+  - [Step 9: Identifying unique/common Tal1 peaks](#step9)
+  - [Additional optional analyses](#stepA)
+* [Concluding remarks](#conclusion)
+* [Useful literature](#literature)
+  
+<a name="introduction"/></a>
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Why did you add this line?

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 6]
```diff
@@ -0,0 +1,420 @@
...
+    <figcaption><b>Figure X:</b> Data can be imported directly with links.</figcaption>
+    
+    
+    <img src="../images/data_uploaded.png"/>   
+    <figcaption><b>Figure X:</b> Imported datasets will appear in the history panel.</figcaption>
+    
+    
+3. In Galaxy, examine the data in a FASTQ file by clicking on the eye icon.
+
+    | :grey_question: Questions |
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Why is it in a box? And if you want to keep the box, why not using several lines for the questions instead of html for the list?

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 7]
```diff
@@ -0,0 +1,420 @@
...
+- *Sophisticated analysis of ChIP-seq data is possible using tools hosted by Galaxy.*
+- *Genomic dataset analyses require multiple methods of quality assessment to ensure that the data are appropriate for answering the biology question of interest.*
+- *By using the sharable and transparent Galaxy platform, data analyses can easily be shared and reproduced.*
+
+## :clap: Congratulations on successfully completing this tutorial!
+
+<a name="literature"/></a>
+# Useful literature
+
+### ChIP-seq
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: I started to collect the useful literature for Chip-Seq in the Chip-Seq README file, to limit redundancy of literature if there is several tutorials. What do you think?

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md [hunk 8]
```diff
@@ -0,0 +1,420 @@
...
+  - **Report only those alignments that \*\*do not\*\* overlap the BED file**: Yes  
+  
+4. Re-name the three files we generated to reflect their contents. 
+
+    | :grey_question: Questions |
+    |:---|
+    | <ul><li>How many Tal1 binding sites are common to both G1E cells and megakaryocytes?</li><li>How many are unique to G1E cells?</li><li>How many are unique to megakaryocytes?</li></ul> |
+
+<a name="stepA"/></a>
+# OPTIONAL ADDITIONAL ANALYSES
```
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Can you remove this part if it is empty? Thanks :)

File: ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md (hunk 1 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md]: This is the previous ChIP-seq tutorial. I'm happy to remove it if we don't want to proceed with filling it out.

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 5 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: These are links marking the various sections of the tutorial that can be jumped to by clicking on the link in the contents table at the top of the tutorial. If there is a more MD-style way to do this, I'll change it!

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 3 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Changed all image references.

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 4 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: While I agree that it might be annoying to update Figure numbers if the tutorial changes, I find it very helpful to have references for each individual figure so that they can be referred to within the tutorial itself, or by someone following the tutorial. Our plan is to change the Xs to numbers once we've finalized the images we want. I vote to keep the captions.

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 6 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: We wanted to have questions "pop out" of the tutorial. Our goal is to force the user to stop and think critically about some of the steps they've just completed, rather than blindly run through all the clicking and parameter setting. We also want to keep the underlying biology at the forefront of the tutorial, and we felt that interspersing the tutorial with questions in a pop-up box was a good choice to do this. I am open to other formatting styles, but this is the best I could come up with without spending too much time messing around :)

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 8 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: I have reorganized the main Steps with some advice from @friedue, and have moved the GC bias section to the Optional Additional Analyses section.

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 7 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: GREAT IDEA! I'll remove this section and link to the README file :)

File: ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md (hunk 1 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md]: I do not know what to do with it now :(

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 5 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: You do not need them. The links are automatics, *e.g.* for `[Introduction](#introduction)` will directly point to Introduction section. You can have a look at the [`CONTRIBUTING.md`](https://raw.githubusercontent.com/bgruening/training-material/master/CONTRIBUTING.md) to have example

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 3 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Great. Thanks

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 4 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Yes I know. We need to think about a good way to do that and limit the human error in figure numbering

File: ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md (hunk 1 above)
Comment (friedue) [inline ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md]: Seeing the comprehensive new tutorial by @malloryfreeberg and @MoHeydarian, I'd vote for removing the old tutorial

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 6 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Yes. It is exactly what I wanted.
I am currently working on a slightly different templating system based on markdown (with not a lot of refactoring of current markdown) to add some pop out and help training. You can have a look here of a proposition: http://bebatut.fr/training-material//RNA-Seq/tutorials/ref_based

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 8 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Do you planned to fill this section then?

Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: So you plan to fill this section?

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 7 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Thanks a lot

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 5 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Cool! Changing now.

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 8 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: I am filling it now :)

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 2 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Will these slides be available soon? It seems like the idea is to link tutorials (like ChIP-seq, RNA-seq) to other tutorials (NGS-QC/mapping) in this collection of training materials. Let me know if I'm wrong. I can remove these broken links but keep the NGS* names as placeholders until the indicated materials are available.

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 4 above)
Comment (malloryfreeberg) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Agreed. It would be nice if there was some auto-increment function in MD...

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 2 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: We are working on it. They will be probably available next week

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 5 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Great! Thanks

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 4 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: I will think about it for the next templating

File: ChIP-Seq/tutorials/TAL1_binding_site_identification.md (hunk 8 above)
Comment (bebatut) [inline ChIP-Seq/tutorials/TAL1_binding_site_identification.md]: Great. Thanks :)

**Comment by bgruening** [comment]
This looks fantastic! @malloryfreeberg please let us know when this is out of WIP!!!!

**Comment by malloryfreeberg** [comment]
Ready for merging!

**Comment by bgruening** [comment]
:tada:  :balloon: very nice addition. Thanks a lot and merry Christmas! :tada:  :balloon:
